changelogs/fragments/780-hotfix-proxmox-kvm-version-6.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+- proxmox_kvm - adapt to PVE 6 version scheme (fixes https://github.com/swayf/proxmoxer/issues/79).
```

```suggestion
- proxmox_kvm - adapt to PVE 6 version scheme (fixes https://github.com/swayf/proxmoxer/issues/79; https://github.com/ansible-collections/community.general/pull/780).
```
We care about Ansible issues/PRs, not that much about issues in unrelated repos :)